fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -1323,6 +1325,18 @@ public void setCboPushDownTopNLimit(long cboPushDownTopNLimit) {
     @VarAttr(name = CBO_DERIVE_RANGE_JOIN_PREDICATE)
     private boolean cboDeriveRangeJoinPredicate = false;
 
+    @VarAttr(name = ADAPTIVE_CHOOSE_EXECUTE_INSTANCES_MODE)
+    private String adaptiveChooseExecuteInstancesMode = SessionVariableConstants.ADAPTIVE_DECREASE;
```

Comment:
Maybe restrict the value to `ADAPTIVE_DECREASE`, `ADAPTIVE_INCREASE`, and `AUTO` in `SetStmtAnalyzer#analyzeSystemVariable`?